extensions/INTEL/SPV_INTEL_fpga_buffer_location.asciidoc

```diff
@@ -0,0 +1,122 @@
+SPV_INTEL_fpga_buffer_location
+==============================
+
+== Name Strings
+
+SPV_INTEL_fpga_buffer_location
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+== Contributors
+
+- Joe Garvey, Intel +
+
+== Notice
+
+Copyright (c) 2023 Intel Corporation.  All rights reserved.
+
+== Status
+
+Final Draft
+
+== Version
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2023-02-01
+| Revision           | 1
+|========================================
+
+== Dependencies
+
+This extension is written against the SPIR-V Specification,
+Version 1.6 Revision 2.
+
+This extension requires SPIR-V 1.0.
+
+== Overview
+
+This extension adds a function parameter decoration that is useful for FPGA targets.  This decoration indicates that a particular global memory pointer can only access a particular physical memory location.  Knowing this information at compile time can allow FPGA compilers to generate load store units of lower area for accesses done through such a pointer.  
+
+== Extension Name
+To use this extension within a SPIR-V module, the following *OpExtension* must be present in the module:
+
+----
+OpExtension "SPV_INTEL_fpga_buffer_location"
+----
+
+== New Capabilities
+This extension introduces a new capability:
+
+----
+FPGABufferLocationINTEL
+----
+
+== Token Number Assignments
+
+--
+[width="40%"]
+[cols="70%,30%"]
+[grid="rows"]
+|====
+|FPGABufferLocationINTEL |5920
+|BufferLocationINTEL     |5921
+|==== 
+--
+
+== Modifications to the SPIR-V Specification, Version 1.6
+
+=== Decoration
+
+Modify Section 3.20, Decoration, adding these rows to the Decoration table:
+
+--
+[options="header"]
+|====
+2+^| Decoration | Extra Operands | Enabling Capabilities
+| 5921 | *BufferLocationINTEL* + 
+Apply only to a pointer. Indicates that the pointer must only point into the physical memory identified by the subsequent literal number operand. The exact semantics of the literal number are implementation defined. | Literal +
```

Comment:
Shouldn't it be:

```suggestion
Apply only to a pointer. Indicates that the pointer and all of its aliases must only point into the physical memory identified by the subsequent literal number operand. The exact semantics of the literal number are implementation defined. | Literal +
```
?

Reply:
I think it makes sense but I am not sure if this is necessary. Wouldn't alias the word itself already implicitly indicates that these two pointers are equivalent?

Reply:
After some thinking - I no longer have objections of leaving this as is.